src/main/java/de/tum/in/www1/exerciseapp/service/CourseService.java

```diff
@@ -60,9 +66,52 @@ public Course save(Course course) {
                 || user.getAuthorities().contains(adminAuthority)
         );
         List<Course> userAuthorizedCourses = userCourses.collect(Collectors.toList());
+        userAuthorizedCourses = moveTutorialToTop(userAuthorizedCourses);
         return userAuthorizedCourses;
     }
 
+    /**
+     * Rearange Course List so Tutorial is always first course
+     *
+     * @return same list reordered so tutorial is first item.
+     */
+    private List<Course> moveTutorialToTop(List<Course> allCourses){
```

Comment:
I am moving the tutorial already here to the top could be done on the client side to, where should it be?